lib/src/reactive_ble.dart

```diff
@@ -50,7 +66,7 @@ class FlutterReactiveBle {
       Repeater(onListenEmitFrom: () async* {
         await initialize();
         yield* _deviceConnector.deviceConnectionStateUpdateStream;
-      }).stream
+      }).stream.asBroadcastStream()
```

Comment:
when testing I found out that this needs to become a broadcast stream btw I can also make it a broadcast repeater

Reply:
If there is already a repeater, I'd go for a broadcast repeater. In this case it doesn't really matter, but the work differently.

Reply:
Hm. For a  repeater to work here, `_deviceConnector.deviceConnectionStateUpdateStream` should either be recreated on every call of `.deviceConnectionStateUpdateStream` getter or be a broadcast stream.

Reply:
hmm good catch indeed let's keep it as is for now this looks the most simplest solution then